### ISSUE TITLE
Information: NgRx method calls with @typescript-eslint/no-unsafe-call rule

### ISSUE THREAD

**Issue by devcrev** [angular-eslint/angular-eslint#182]
Code like this was triggering warnings/errors with the rule @typescript-eslint/no-unsafe-call rule. One for the `createReducer` call and another for the `on` call.

```typescript
export const technicianReducer = createReducer(
    initialState,

    on(
        TechnicianActions.sanityCheck,

        (state, action) => ({
            ...state,
            sanity: action.sanityValue
        })
    )
);
```

Updating the npm packages `@typescript-eslint/eslint-plugin` and `@typescript-eslint/parser` to the latest version (4.7.0 for both of them at the time of this writing) resolved the issue.

Running the `ng add @angular-eslint/schematics` command installs 4.3.0 of the aforementioned packages.

**Comment by JamesHenry** [comment]
Thanks @devcrev! Do you fancy opening a PR to bump the versions everywhere they are referenced?

**Comment by devcrev** [comment]
It appears that `@typescript-eslint/eslint-plugin` and `@typescript-eslint/parser` version numbers are chosen from the version number assigned to `@typescript-eslint/experimental-utils` in `packages/schematics/package.json`. In this case, what the repo looks like today, i.e., "4.3.0". There are references, in other files to the `experimental-utils` as well as the counterpart `parser`. I suspect things are more involved than replacing version numbers where found.

In my single project Angular app all I did was add the `^` character in front of the version numbers (`package.json`) and updated with `npm update {package}`

Before:
```
    "@typescript-eslint/eslint-plugin": "4.3.0",
    "@typescript-eslint/parser": "4.3.0",
```

After my change:
```
    "@typescript-eslint/eslint-plugin": "^4.3.0",
    "@typescript-eslint/parser": "^4.3.0",
```

Then npm update made these changes:
```
    "@typescript-eslint/eslint-plugin": "^4.7.0",
    "@typescript-eslint/parser": "^4.7.0",
```

**Comment by devcrev** [comment]
@JamesHenry, you may dismiss this issue. I think my IDE was slow to lint after I imported my NgRx reducer into my app.module.ts. The problem does not exist with 4.3.0 of the aforementioned NPM packages. My IDE reports no issues and running `ng lint` shows no problems either.

BTW - nice progress made with the release of 0.8.0-beta.1 compared to the last release!
